spotinst/version.go

```diff
@@ -1,7 +1,7 @@
 package spotinst
 
 // SDKVersion is the current version of the SDK.
-const SDKVersion = "1.53.1"
+const SDKVersion = "1.54.0"
```

Comment:
```suggestion
const SDKVersion = "1.53.1"
```

We bump the version number before creating a new release.